extensions/mssql/extension.webpack.config.js

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+//@ts-check
+
+'use strict';
+
+const withDefaults = require('../shared.webpack.config');
+
+module.exports = withDefaults({
+	context: __dirname,
+	entry: {
+		extension: './src/main.ts'
+	},
+	externals: {
+		'clipboardy': 'commonjs clipboardy',
+		'dataprotocol-client': 'commonjs dataprotocol-client',
+		'vscode-languageclient': 'commonjs vscode-languageclient'
```

Comment:
FYI we also use the request module here. Should we make it an external like in azurecore?

Reply:
Just checked - Gene put it in the wrong package.json. I'll have a PR out that fixes insiders build ASAP, please update this once it's in.